R/utils_cypress.R

```diff
@@ -68,7 +83,10 @@ create_node_list <- function(tests_path, port) {
       "performance-test" = glue(
         "start-server-and-test run-app http://localhost:{port} run-cypress"
       ),
-      "run-app" = glue("cd root && Rscript -e 'shiny::runApp(port = {port})'"),
+      "run-app" = glue(
+        "cd root && ",
```

Comment:
I think the && is not needed in this case (I am not sure)

Reply:
The change here was using " instead of ' inside the string. Windows on my machine had a problem with this.

@DouglasMesquita do you suggest using ";" instead ? "&&" 

ps. I find it funny that windows doesn't have a problem with "cd" when calling node (the original exception with R code)

Reply:
I tried it and it fails on windows! I think it needs to be "&&", unless you have another suggestion